api_app/core/config.py

```diff
@@ -26,6 +26,7 @@
 STATE_STORE_DATABASE = "AzureTRE"
 STATE_STORE_RESOURCES_CONTAINER = "Resources"
 STATE_STORE_RESOURCE_TEMPLATES_CONTAINER = "ResourceTemplates"
+STATE_STORE_RESOURCES_HISTORY_CONTAINER = "ResourcesHistory"
```

Comment:
up to debate.
ResourceTemplates --> implies --> ResourceHistories?

Reply:
after a discussion ResourceHistory was picked